src/coreclr/jit/inlinepolicy.cpp

```diff
@@ -1628,7 +1628,7 @@ double ExtendedDefaultPolicy::DetermineMultiplier()
         //
         //  void Caller() => DoNothing(42); // 42 is going to be boxed at the call site.
         //
-        multiplier += 0.5;
+        multiplier += 0.5 * m_ArgIsBoxedAtCallsite;
```

Comment:
I suppose it's hard to check what the impact of this change is on its own.

Reply:
I had it much more aggressive and it caused some gratuitous size regressions. This seems like a reasonable compromise -- most of the size regressions now are from exposed stack allocated objects.